src/ol/layer/STAC.js

```diff
@@ -466,6 +470,25 @@ class STACLayer extends LayerGroup {
 
     const sources = [];
     switch (link.rel) {
+      case 'pmtiles':
+        const p = new pmtiles.PMTiles(link.href);
+        const headers = await p.getHeader();
+        let source;
+        switch (headers.tileType) {
+          case pmtiles.TileType.Mvt:
+            source = new PMTilesVectorSource(options);
+            break;
+          case pmtiles.TileType.Avif:
+          case pmtiles.TileType.Jpeg:
+          case pmtiles.TileType.Png:
+          case pmtiles.TileType.Webp:
+            source = new PMTilesRasterSource(options);
+            break;
+          default:
+            return; // Unsupported
```

Comment:
how is this case exposed to user? use case for TileType.Unknown is sticking your own arbitrary gridded data into tiles, but I'm not sure how it can be visualized on the web if it doesn't conform to one of the known types

Reply:
This means it just doesn't create a layer for the PMTiles in OL STAC. It's ignored due to the exact reasons that you are mentioning.